Product/pre_process.js

```diff
@@ -0,0 +1,59 @@
+import { ElectronicProduct } from './ElectProductClass.js';
+import { Product } from './ProductClass.js';
+
+import * as ProductService from './ProductService.js'
+import * as pre_process from './pre_process.js'
+
+
+//서버에서 받아온 response를 전자제품인지 아닌지에 따라 다른 배열로 나눔
+export async function getProductarray(response){
+    let products = [];
+    let elec_products = [];
+    // console.log("count of response[list]", response["list"].length)
+    for (let product of response["list"]){
+        if (product.tags.includes == "전자제품" || product.tags.includes =='전자제품'){
+            elec_products.push(product);
+        }else{
+            products.push(product);
+        }
+    }
+    return [products, elec_products];
```

Comment:
이렇게 작성하면 이중배열로 리턴하게 되는데, 이는 좋은 방식은 아닌 것 같아요. 차라리 객체로 작성해서 리턴하는 것이 좋아보입니다! 이중배열같은 다중배열은 성능저하의 원인도 있고, 사용자가 인지하기 힘든 문제가 있어요.

```suggestion
    return {products, elec_products};
```